01f_work_with_vector_int.cpp

```diff
@@ -33,15 +33,36 @@ void print_stats(const std::vector<int> input_object)
     print_vector(input_object);
 }
 
+std::vector<int> vectors_sum(std::vector<int> argument1, std::vector<int> argument2)
+{
+    std::vector<int> sum_vector;
+    for (int i=0; i < std::max(argument1.size(), argument2.size()); i++)
+    {
+        sum_vector.push_back(argument1[i] + argument2[i]);
+    }
+    return sum_vector;
+}
+
+
 int main()
 {
-    std::vector<int> input1(16, 42);
+    const int vector_size = 10;
+    std::vector<int> input1(vector_size, 42);
+    std::vector<int> input2(vector_size, 53);
 
-    print_stats(input1);
-    input1[2] = 5;
-    print_vector(input1);
-    input1.push_back(99);
+    for (int i=0; i < std::max(input1.size(), input2.size()); i++){
+        if (i%2 == 0)
+        {
+            input1[i] += i;
+        }
+        else
+        {
+            input2[i] -= i;
+        }
+    }
     print_vector(input1);
+    print_vector(input2);
+    print_vector(vectors_sum(input1, input2));

```

Comment:
Please do not use several operations at single line. It has many disadvantages.

Reply:
fixed by https://github.com/Olgerd96/cpp/commit/d64aa2d1f9f11396ac84ec0dc6000d8a386b3f9b